ts-sdk/whirlpool/tests/createPool.test.ts

```diff
@@ -1,5 +1,263 @@
-import { describe } from "vitest";
+import { describe, it, beforeAll } from "vitest";
+import {
+  createSplashPoolInstructions,
+  createConcentratedLiquidityPoolInstructions,
+} from "../src/createPool";
+import {
+  DEFAULT_FUNDER,
+  setDefaultFunder,
+  SPLASH_POOL_TICK_SPACING,
+} from "../src/config";
+import { setupMint } from "./utils/token";
+import { setupMintTEFee } from "./utils/tokenExtensions";
+import { orderMints } from "../src/token";
+import { rpc, sendTransaction, signer } from "./utils/mockRpc";
+import { fetchMaybeWhirlpool } from "@orca-so/whirlpools-client";
+import assert from "assert";
+import type { Address } from "@solana/web3.js";
+import { assertAccountExists, lamports } from "@solana/web3.js";
+import { _TICK_ARRAY_SIZE, priceToSqrtPrice } from "@orca-so/whirlpools-core";
 
-describe.skip("Create Pool", () => {
-  // TODO:
+describe("Create Pool", () => {
+  let mintA: Address;
+  let mintB: Address;
+
+  beforeAll(async () => {
+    const mint1 = await setupMint();
+    const mint2 = await setupMint();
+    [mintA, mintB] = orderMints(mint1, mint2);
+  });
+
+  it("Should throw an error if funder is not set", async () => {
+    setDefaultFunder(DEFAULT_FUNDER);
+    await assert.rejects(
+      createConcentratedLiquidityPoolInstructions(rpc, mintA, mintB, 64, 1),
+    );
+    setDefaultFunder(signer);
+  });
+
+  it("Should throw an error if token mints are not ordered correctly", async () => {
+    await assert.rejects(
+      createConcentratedLiquidityPoolInstructions(rpc, mintB, mintA, 64, 1),
+    );
+  });
+
+  it("Should create splash pool", async () => {
+    const price = 10;
+    const sqrtPrice = priceToSqrtPrice(price, 6, 6);
+
+    let signerAccount = await rpc.getAccountInfo(signer.address).send();
```

Comment:
Think nicer if we implement getBalance on the mockRpc so we don't have to duplicate this every time.